app/components/Texting/Texting.scss

```diff
@@ -0,0 +1,175 @@
+@import '../../styles/utils/_helpers.scss';
+
+.overlay {
+  position: fixed;
+  height: 100%;
+  width: 100%;
+  top: 70px;
+  background: rgba(0, 0, 0, 0.2);
+  @media screen and (max-width: 900px ){
+    top: 50px;
+  }
+}
+
+.content {
+  position: relative;
+  width: 500px;
+  height: 660px;
+  background: #FFFFFF;
+  margin: 35px auto;
+  padding: 30px;
+  box-shadow: 0 3px 8px rgba(50, 50, 50, 0.17);
+  box-sizing: border-box;
+  border-radius: 8px;
+
+  @media screen and (max-width: 900px ) {
+    width: 100%;
+    height: 100%;
+    font-size: 14px;
+    padding: 20px;
+    margin: 0 auto;
+  }
+}
+
+.closeButton {
+  background: none;
+  border: 0;
+  cursor: pointer;
+  display: inline-block;
+  padding: 0;
+  position: absolute;
+  right: 20px;
+  top: 20px;
+}
+
+.title{
+  margin-top: 1.5em;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 1.8em;
+}
+
+.description {
+  font-size: 1em;
+  margin-top: 2em;
+}
+
+.input {
+  background-color: $color-white;
+  height: 50px;
+  @media screen and (max-width: 900px ) {
+    height: 35px;
+  }
+}
+
+.label {
+  font-size: 1em;
+  color: $color-grey5;
+  margin-top: 2em;
+  display: block;
+}
+
+.dataRates {
+  font-size: 0.9em;
+  color: $color-grey5;
+}
+
+.checkBox {
+  margin-top: 1em;
+  font-size: 0.9em;
+  color: $color-grey5;
+  align-items: center;
+}
+
+.checkBox input[type="checkbox"] {
+  margin-right: 10px;
+  width: auto;
+  cursor: pointer;
+}
+
+.button {
+  font-size: calc-em(16px);
+  padding: calc-em(7px) calc-em(15px);
+  background: $color-brand;
+  color: $color-white;
+  border-radius: 3px;
+  margin-left: 20px;
+
+  @media screen and (max-width: 900px ) {
+    width: 100%;
+    font-size: 14px;
+    margin: 8px auto;
+  }
+  &.disabled {
+    background: $color-brand-light;
+  }
+  &.cancel {
+    border: 1px solid $color-brand;
+    color: $color-brand;
+    background: $color-white;
+    @media screen and (max-width: 900px ) {
+      display: none;
+    }
+    &:hover {
+      background: $color-brand;
+      color: $color-white;
+    }
+  }
+}
+
+.buttons {
+  display: flex;
+  margin-top: 2em;
+  justify-content: flex-end;
+  @media screen and (max-width: 900px ) {
+    margin-top: 0;
+  }
+}
+.privacy {
+  margin-top: 2em;
+  font-size: 14px;
+  text-align: center;
+  @media screen and (max-width: 900px ) {
+    margin-top: 1em;
+    font-size: 12px;
+  }
+}
+
+.circle {
+  width: 150px;
+  height: 150px;
+  background: #00916A;
+  border-radius: 50%;
+  text-align: center;
+}
+
+.successDiv {
+  position: relative;
+  text-align: center;
+  top: 33%;
+  left: 35%;
+}
+
+.responseIcon {
+  width: 116.67px;
+  margin-top: 20%;
+}
+
+.successText {
```

Comment:
This font color also looks like it should be darker, an absolute black rather than a dark gray.